src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1567,7 +1583,17 @@ private function deleteProperty(string $path): void
             // no we really don't know that path
             throw new ItemNotFoundException('No item found at '.$path);
         }
+        $this->removePropertiesFromNode($nodeId, [$path]);
+    }
 
+    /**
+     * Removes a list of properties from a given node
+     *
+     * @param string|int $nodeId
+     * @param array<string> $paths  Path belonging to that node that should be deleted
+     */
+    private function removePropertiesFromNode(string|int $nodeId, array $paths): void
```

Comment:
php 8.1? Is that okay or do we need backporting to older versions?

Reply:
the upcoming 2.0 version of jackalope is targetting 8.0

8.0 is already old again, but lets stick with it for now. meaning you have to remove the type declaration in the signature and we rely on the phpdoc typehint.

Reply:
the 8.0 tests are happy. are you sure this is not allowed with php 8.0?